doc/api/replication.md

```diff
@@ -0,0 +1,106 @@
+# Thali Cordova Peer Communication API #
+
+## Methods:
+- `StartBroadcasting`
+- `StopBroadcasting`
+- `Connect`
+- `Disconnect`
+- `KillConnection`
```

Comment:
Since KillConnection is never supposed to be used by 'normal' code it would be good to put KillConnection into a completely separate section with a black box warning that any use of it outside of testing scenarios will result in electrocution and/or uncontrolled vomiting